src/vs/workbench/services/preferences/browser/preferencesService.ts

```diff
@@ -313,7 +313,11 @@ export class PreferencesService extends Disposable implements IPreferencesServic
 				editor.search(options.query);
 			}
 		}
+	}
 
+	async openNotifications(): Promise<void> {
+		const options = { pinned: true, revealIfOpened: true };
```

Comment:
Here if I try to instantiate the editor directly (rather than do that via the input, it doesn't work)